scripts/postcss/ThemePlugin.js

```diff
@@ -0,0 +1,58 @@
+const fs = require('fs-extra');
+const path = require('path');
+const postcss = require('postcss');
+
+const CONFIG = 'terra-theme.config.js';
+const SUPPORTED_THEMES = [
+  'orion-fusion-theme',
+  'clinical-lowlight-theme',
+  'cerner-clinical-theme',
+];
+
+/**
+ * The purpose of this plugin is to create a default theme from a scoped theme
+ * and to remove any supported themes that are not desired.
+ */
+module.exports = postcss.plugin('postcss-test-plugin', (config) => {
+  // Retrieve theme config.
+  let themeConfig = {};
+  if (config) {
+    themeConfig = config;
+  } else {
+    const defaultConfig = path.resolve(process.cwd(), CONFIG);
+    if (fs.existsSync(defaultConfig)) {
+      // eslint-disable-next-line global-require, import/no-dynamic-require
+      themeConfig = require(defaultConfig);
+    }
+  }
+
+  // Add the . to find the selector.
+  const defaultThemeSelector = `.${themeConfig.theme}`;
+
+  const scopedThemes = themeConfig.scoped || [];
+
+  // Find the set of known themes that should not be included.
+  const themesToRemove = SUPPORTED_THEMES.reduce((acc, theme) => {
+    if (!scopedThemes.includes(theme)) {
+      acc.push(`.${theme}`);
+    }
+    return acc;
+  }, []);
+
+  return (root) => {
+    if (defaultThemeSelector || themesToRemove.length > 0) {
```

Comment:
not a big deal, but i believe a length existence check does the same thing
```suggestion
    if (defaultThemeSelector || themesToRemove.length) {
```